src/sentry/models/organization.py

```diff
@@ -201,6 +201,9 @@ class flags(TypedClassBitField):
         # Prevent superuser access to an organization
         prevent_superuser_access: bool
 
+        # Disable org-members from inviting members
+        disable_member_invite: bool
```

Comment:
do we want disable to be true or false by default? the default for a bitfield is false

Reply:
@cathteng by default we want members to invite other members, so we want this flag to be false by default

Reply:
yes, default should be false.

Reply:
We have a feature flag for launching it gradually but default is false unless they turn it to true to disable.